index.html

```diff
@@ -0,0 +1,339 @@
+        
+      </section>
+
+      <section class="main-section" id="Function_declarations">
+        <header>Function declarations</header>
+        <div>
+          <p>A function definition (also called a function declaration, or function statement) consists 
+            of the function keyword, followed by:</p>
+          <ul>
+            <li>The name of the function.</li>
+            <li>A list of arguments to the function, enclosed in parentheses and separated by commas.</li>
+            <li>The JavaScript statements that define the function, enclosed in curly brackets, { }.</li>
+          </ul>
+          <p>For example, the following code defines a simple function named square:</p>
+          <code>function square(number) { return number * number; }</code>
+          <p>The function square takes one argument, called number. The function consists of one statement 
+            that says to return the argument of the function (that is, number) multiplied by itself. The 
+            return statement specifies the value returned by the function.</p>
+          <code>return number * number;</code>
+          <p>Primitive parameters (such as a number) are passed to functions by value; the value is passed 
+            to the function, but if the function changes the value of the parameter, this change is not 
+            reflected globally or in the calling function.</p>
+        </div>
+        
+      </section>
+
+      <section>
+        <header>Reference</header>
+        <ul>
+          <li>All the documentation in this page is taken from <a href=
+            "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide">MDN</a></li>
+        </ul>
+      </section>
+    </main>
+    
+    <nav id="navbar">
+      <header>JS Documentation</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li><a class="nav-link" href="#What_you_should_already_know">What you should already know</a></li> 
+        <li><a class="nav-link" href="#JavaScript_and_Java">JavaScript and Java</a></li>
+        <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+        <li><a class="nav-link" href="#Variables">Variables</a></li>
+        <li><a class="nav-link" href="#Declaring_variables">Declaring variables</a></li>
+        <li><a class="nav-link" href="#Variable_scope">Variable scope</a></li>
+        <li><a class="nav-link" href="#Global_variables">Global variables</a></li>
+        <li><a class="nav-link" href="#Global_variables">Global variables</a></li>
+        <li><a class="nav-link" href="#Constants">Constants</a></li>
+        <li><a class="nav-link" href="#Data_types">Data types</a></li>
+        <li><a class="nav-link" href="#if_else_statement">if...else statement</a></li>
+        <li><a class="nav-link" href="#while_statement">while statement</a></li>
+        <li><a class="nav-link" href="#Function_declarations">Function declarations</a></li>
+     </ul>
+    </nav> 
```

Comment:
[Optional]. Your nav element should appear before the main tag for a better HTML structure.😄